nginx.template

```diff
@@ -20,6 +20,13 @@ http {
     # Changes based on whether running on kubernetes or plain docker
     include '/nginx-resolver.conf';
 
+    gzip on;
+    gzip_vary on;
+    gzip_min_length 10240;
```

Comment:
Noen grunn til at vi kun vil komprimere responser over 10kb? Ser ut til at vi må legge på `Content-Length` parametere i apiene om den skal bli respektert.
["The length is determined only from the “Content-Length” response header field." ](https://nginx.org/en/docs/http/ngx_http_gzip_module.html#gzip_min_length)

Reply:
Fant koden på ei side. Det skulle være 1024. Tanken var vel at små filer kan bli større ved gzipping.

Reply:
Men kan ta den bort om vi ikkje har content-length.

Reply:
Vi kunne jo evt lagt til content-length headere om det er en fordel, jeg var bare nysgjerrig 😊